components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/api/dto/Attributes.java

```diff
@@ -16,15 +16,23 @@
  */
 package org.apache.camel.component.salesforce.api.dto;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.thoughtworks.xstream.annotations.XStreamAsAttribute;
+
 public class Attributes extends AbstractDTOBase {
 
     private String type;
     private String url;
 
+    @XStreamAsAttribute
+    String referenceId;
+
+    @JsonProperty("type")
```

Comment:
Probably not needed, doesn't hurt if it's here though, same for the setter annotation.
```suggestion
```